File: 백준/Bronze/9295. 주사위/주사위.cc
#include <iostream>
using namespace std;

int main() {

	int n;
	cin >> n;

	for (int i = 0; i < n; i++)
	{
		int sum = 0;
		int d1, d2;
		cin >> d1 >> d2;

		sum = d1 + d2;

		cout << "Case " << (i + 1) << ": " << sum << endl;
	}
}

File: 백준/Bronze/9295. 주사위/주사위.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            int d1 = sc.nextInt();
            int d2 = sc.nextInt();
            int sum = d1 + d2;

            System.out.println("Case " + (i+1) + ": " + sum);
        }
    }
}